try {
	var a = 0;
}catch (Error){
	   return 1;
}finally{
		return 0;
}

try{}finally{}
try{}catch(Error){}
